.github/workflows/localization.yml

```diff
@@ -14,11 +14,40 @@ jobs:
         TDBUILD_AAD_APPLICATION_CLIENT_SECRET: ${{ secrets.TDBUILD_AAD_APPLICATION_CLIENT_SECRET }}
       run: ./localize.sh
     - name: Create Pull Request
-      uses: peter-evans/create-pull-request@v3.5.0
-      with:
-        commit-message: Latest localized resource files from Touchdown Build
-        title: '[Localization] Localized Resource Files'
-        body: 'Latest localized resource files from Touchdown Build'
-        branch: tdbuild
-        labels: Automated PR
-        branch-suffix: timestamp
+      env:
+        GITHUB_TOKEN: ${{ secrets.GH_PAT }}
+      run: |
+        set -e
+        echo "Start."
+        # Configure git and Push updates
+        git config --global user.email pyeruva@microsoft.com
```

Comment:
Is it possible to use some variable that gets populated at the run time or any other generic email?